config/locales/en/lookup.yml

```diff
@@ -10,8 +10,10 @@ en:
         input_error: Enter a valid postcode
         error_description: Enter a valid postcode
     lookup:
-      input_label: Enter your postcode
-      hint_text: For example LS1 1UR
+      input_label: Enter a full postcode 
+      hint_text: |
+        We use this to give you more accurate information about the area.<br/>
+        For example, LS1 1UR
```

Comment:
This doesn't work, I'm afraid:

<img width="1308" alt="Screenshot 2020-08-27 at 09 30 20" src="https://user-images.githubusercontent.com/5793815/98360849-76db6a00-2022-11eb-967f-7f6c1cba77de.png">

I tried it with a `<p>` instead of the `<br/>` and that didn't work either.

cc @DilwoarH